web/src/composebox_typeahead.js

```diff
@@ -1187,7 +1188,7 @@ export function initialize() {
             return this;
         },
         header() {
-            return "<em>Start a new topic or select one from the list.</em>";
+            return render_topic_typeahead_hint;
```

Comment:
Updated! Just wondering if this is a code style thing since when I did manual testing on it, it seems like work just fine.

Reply:
It's very possible that the typeahead library is clever and checks if it has a function rather than a string and calls it if so. If we wanted to take advantage of that, though, there's probably a better way to write it that doesn't involve defining this stub function.